HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+    "RequireDigit": "false",
+    "RequiredLength": "6",
+    "RequireNonAlphanumeric": "false",
+    "RequireUppercase": "false",
+    "RequireLowercase": "true"
+  },
+  "MailConfig": {
+    "Port": 587,
+    "UseSsl": "true",
+    "PickupDirectoryLocation": "<AbsolutePathToFolderWhereTheEmailsWillBeStored>"
+  },
+  "UserApiOptions": {
+    "RestrictedRoutes": [
+      "/api/private"
+    ],
+    "PublicRoutes": [
+      "/api/attachment",
+      "/api/public"
+    ]
+  },
+  "ClientFilteringConfig": {
+    "CaseSensitive": "false",
+    "AccentSensitive": "false"
+  },
+  "LanguageConfig": {
+    "Default": "en-US",
+    "Allowed": [
+      {
+        "Culture": "en-US",
+        "Caption": "English",
+        "ResetPasswordMailSubject": "Reset Password",
+        "ResetPasswordMailBodyFileName": "testResetPwFile.txt"
+      }
+    ]
+  }
+}
+```
+Make sure you set 
+- **PathToClientApp** to the absolute path to the
+`frontend-html\dist` folder where we built the frontend application before.
+- **PathToCustomAssetsFolder** to absolute path to a folder where you keep the application images. The images don't have to be there for development.
+- **PickupDirectoryLocation** to absolute path to a folder where the emails will be saved instead 
+of sending them to the actual email addresses. This is quite useful for development.
+- **PasswordForJwtCertificate** password you entered when creating the `serverCore.pfx` 
+file
+
+All back slashes in the paths should be escaped i.e. \\ instead of \.
+
+When you opened the solution Visual Studio created the file 
+```
+backend\Origam.Server\Properties\launchSettings.json
+```
+Open the file and set the **https** port to **44357**. You can ignore the http port number.
+
+
+
+Now go to Visual Studio and run the Origam.Server project. Everything should work and 
+your default browser should open with the login screen. Enter the following:
+
+**Username: testUser**
+
+**Password: 9ECZJ83w4UNQPjrR**
+
+The production build of the client application located at `frontend-html\dist` will run after you login.
+
+### Run the Javascript Development Server
+Next we will run the front end javascript application in a development server so that
+we can debug it. 
+Just open CMD.exe in the `frontend-html` folder and run
+```
+yarn dev
+```
+The development server will start and show you the port number where you can reach it.
+
+### Conclusion 
+We have configured the Origam development environment. Now you can debug all parts of Origam.
+
+To debug the Architect in Visual Studio set the startup project to **OrigamArchitect** and switch the solution configuration to **Debug Architect**. Then run the project.
+
+If you wan to debug the server set the startup project to **Origam.Server** switch the solution configuration to **Debug Server** and run the project.
+
+If you need to debug the front end application in browser run the Origam server (previous paragraph) and start the node development server with `yarn dev`. 
```

Comment:
After reading from the beginning to the end I would just wonder for a while about why I had to do all these steps manually. But this is just a caustic side-note :)